aws/studio_hpo_container_setup.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Delete image, we can also link to the SageMaker HPO demo for this additional detail</p><p> </p><p>I'll go through the rest of this notebook after, but continue going through and cleaning up to just keep specific container build details</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>